views/islandora_usage_stats.views.inc

```diff
@@ -67,6 +67,75 @@ function islandora_usage_stats_views_data() {
       ),
     ),
   );
+  $data['islandora_usage_stats_datastreams'] = array(
+    'table' => array(
+      'group' => t('Islandora Usage Stats'),
+      'base' => array(
+        'field' => 'id',
+        'title' => t('Islandora Usage Stats Datastreams'),
+        'help' => t('Datastream information.'),
```

Comment:
Oh and **help** doesn't seem to appear, at least not in the Garland theme. I just get a select box to choose my data table with the titles.